scripts/development/.env_example

```diff
@@ -74,6 +74,11 @@ RPC_PASSWORD=s750SiJnj50yIrmwxPnEdSzpfGlTAHzhaUwgqKeb0G1j
 API_PORT=8080
 API_HOST=localhost
 BLOCKCHAIN_LOG_LEVEL=trace
+NODE_ENV=development
+#CERT_PATH=""
+#CERT_CA_PATH=""
+#CERT_KEY_PATH=""
```

Comment:
Make a comment that shortly explains this vars and why they are  normally not needed